AsyncDisplayKit/ASMapNode.mm

```diff
@@ -161,7 +160,18 @@ - (MKCoordinateRegion)region
 
 - (void)setRegion:(MKCoordinateRegion)region
 {
-  self.options.region = region;
+  MKMapSnapshotOptions * __weak oldOptions = self.options;
+  MKMapSnapshotOptions * options = [[MKMapSnapshotOptions alloc] init];
+  options.camera = oldOptions.camera;
+  options.mapRect = oldOptions.mapRect;
+  options.mapType = oldOptions.mapType;
+  options.showsPointsOfInterest = oldOptions.showsPointsOfInterest;
+  options.showsBuildings = oldOptions.showsBuildings;
+  options.size = oldOptions.size;
+  options.scale = oldOptions.scale;
+  options.region = region;
+  self.options = options;
```

Comment:
I agree, but I thought that since you're only changing the region, we shouldn't change the configuration that the developer might have had. I suspect that most of this will be left as default values and thus will be irrelevant.

I think the most important ones are the showsPointsOfInterest and showsBuildings, I'm not sure we need the rest, I just put them in for completion's sake.